boa3/internal/analyser/supportedstandard/standardanalyser.py

```diff
@@ -69,61 +69,85 @@ def _validate_standards(self):
         try:
             for standard in self.standards:
                 if standard in supportedstandard.neo_standards:
-                    current_standard = supportedstandard.neo_standards[standard]
-
-                    # validate standard's methods
-                    for standard_method in current_standard.methods:
-                        method_id = standard_method.external_name
-                        is_implemented = False
-
-                        found_methods = self.get_methods_by_display_name(method_id)
-                        for method in found_methods:
-                            if isinstance(method, Method) and current_standard.match_definition(standard_method, method):
-                                is_implemented = True
+                    standard_is_correct = False
+                    errors = []
+
+                    for current_standard in supportedstandard.neo_standards[standard]:
+                        check_next_standard = False
+
+                        # validate standard's methods
+                        for standard_method in current_standard.methods:
+                            method_id = standard_method.external_name
+                            is_implemented = False
+
+                            found_methods = self.get_methods_by_display_name(method_id)
+                            for method in found_methods:
+                                if isinstance(method, Method) and current_standard.match_definition(standard_method, method):
+                                    is_implemented = True
+                                    break
+
+                            if not is_implemented:
+                                errors.append(CompilerError.MissingStandardDefinition(standard, method_id, standard_method))
+                                check_next_standard = True
```

Comment:
why does it set `check_next_standard = True` here after appending the error?
can an error be logged when the next standard on the array is implemented correctly?